corgi/core/constants.py

```diff
@@ -144,7 +144,7 @@
             ON ("core_component_productstreams"."productstream_id" = "core_productstream"."uuid")
             {LATEST_FILTER_WHERE}
             AND (include_inactive_streams OR core_productstream.active)
-            AND (ps_ofuri IS NOT NULL AND core_productstream.ofuri = ps_ofuri);
+            AND (ps_ofuris IS NOT NULL AND core_productstream.ofuri = ANY(ps_ofuris));
```

Comment:
I guess the test_latest_components_by_streams_filter_with_multiple_products test is now failing because it returns returns the latest package in any of the streams. In the test I assume it would now return the package with release '11' which is only part of the rhel-8 stream. 

Unfortunately I don't think that is acceptable because there might be a problem with releases prior to '11', which does not affect release '11'. Users need to know that rhel-7 ships release '10' as it's latest in that case. I think the only way to do this properly is to loop on all the ps_ofuris and calculate latest packages for each of them.

Reply:
correct, I am mid stream pushing this loop into db .... if we can wait will be done by tomorrow

Reply:
I think that given it is Thursday (or Friday) already we could wait for this. Since we try not to release right before a weekend?